packages/docs/src/components/CodepenEdit.vue

```diff
@@ -50,7 +50,6 @@ export default {
             return JSON.stringify({
                 title: `${this.$route.meta.title} ${this.title ? this.title.toLowerCase() : ''} - Buefy example`,
                 tags: ['buefy', 'vue', 'bulma'],
-                editors: this.style ? 111 : 101,
```

Comment:
To keep the current behavior, we have to leave `editors: 101`.

Reply:
Btw, the meaning of `editors`. <https://blog.codepen.io/documentation/prefill/#:~:text=editors%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3A%20%22101%22%2C%20//%20Set%20which%20editors%20are%20open.%20In%20this%20example%20HTML%20open%2C%20CSS%20closed%2C%20JS%20open>

Reply:
i.e., equivalent to HTML open, CSS closed, JS open